hotness/tests/test_consumer.py

```diff
@@ -0,0 +1,130 @@
+"""
+Unit tests for hotness.consumer
+"""
+from __future__ import unicode_literals, absolute_import
+
+import unittest
+
+import mock
+
+import fedmsg.config
+
+from hotness import consumers
+
+
+class TestConsumer(unittest.TestCase):
+    def setUp(self):
+        self.bz = mock.patch('hotness.bz.Bugzilla')
+        self.bz.__enter__()
+        self.koji = mock.patch('hotness.buildsys.Koji')
+        self.koji.__enter__()
+
+        test_config = fedmsg.config.load_config()
+        test_config["hotness.cache"] = {
+            "backend": "dogpile.cache.null",
+        }
+        test_config['hotness.requests_retries'] = 0
+
+        class MockHub(mock.MagicMock):
+            config = test_config
+
+        self.consumer = consumers.BugzillaTicketFiler(MockHub())
+        import logging
```

Comment:
Please have all imports at the top of the module

Reply:
Moved in b3de423.